deck.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'model'
+require 'pry'
+
+class Deck
+  attr_reader :flashcards_from_file, :flashcards, :definition
+  attr_accessor :term
+
+  def initialize(flashcards_from_file)
+    @flashcards = []
+    flashcards_from_file.each_with_index do |element, index|
+      if index % 2 == 0
+        self.flashcards << Card.new({definition: element})
+      else
+        self.flashcards.last.term = element
+      end
+    end
+    @flashcards
+  end
+
+  def move_card_to_back
+    flashcards.push(flashcards.shift)
+  end
+
+  def generate_random_card
+    flashcards.shuffle
+  end #?
```

Comment:
The name of the method `generate_random_card` makes me think it will return a card object. Instead, this simply shuffles the deck.

There's no harm in calling this method "shuffle". That way the Deck would have a shuffle method which simply calls the shuffle method on the cards array it is encapsulating.